orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/AdminController.groovy

```diff
@@ -74,6 +87,35 @@ class AdminController {
     forceExecutionCancellationCommand.forceCancel(executionType, executionId, canceledBy)
   }
 
+
+  @PostMapping(value = "/executions/")
+  @ResponseStatus(HttpStatus.CREATED)
+  Map<String, String> createExecution(@RequestBody Execution execution) {
+
+    if (front50Service && !front50Service.get(execution.application)) {
```

Comment:
I believe `front50Service.get()` will raise an exception if the application doesn't exist (vs. returning null).

I'm not sure we want to import executions for applications that don't exist?